db/import_mb_data.py

```diff
@@ -366,10 +386,33 @@ def load_gender(connection, gids_in_AB):
     return MB_gender_data
 
 
-def load_area(connection, gids_in_AB):
+def load_area(connection, gids_in_AB, MB_artist_data):
```

Comment:
More details needed about what exactly is `MB_artist_data` here

Reply:
Updated the docstrings for all the functions specifying every argument and return value.